tests/ClientTest.php

```diff
@@ -46,6 +46,64 @@ public function testGetSupportedEndpoints()
         $this->assertContains('wikipediaSearch', $endpoints);
     }
 
+    public function testGetLastTotalResultsCount()
```

Comment:
Looks like this test should be split into a couple of smaller ones, each having the appropriate title (per your comments).